src/mapping.ts

```diff
@@ -46,6 +47,33 @@ export function handleTransferSingle(event: TransferSingle): void {
   nft.save();
 }
 
+export function handleTransferKnowOrigin(event: KnowOriginTransferEvent): void {
+  let address = event.address.toHexString();
+  if (NftContract.load(address) == null) {
+    let nftContract = new NftContract(address);
+
+    nftContract.name = fetchName(event.address);
+    nftContract.symbol = fetchSymbol(event.address);
+    nftContract.platform = "KnowOrigin";
+    nftContract.save();
+  }
+
+  let id = event.transaction.hash.toHex() + "-" + event.logIndex.toString()
+  let contract = ERC721.bind(event.address);
+  let nft = Nft.load(id);
+  if (nft == null) {
+    nft = new Nft(id);
+    nft.contract = event.address.toHexString();
+    nft.tokenID = event.params._tokenId;
+    nft.creatorName = contract.nameOfCreator();
```

Comment:
`nameOfCreator` function was removed from `ERC721` ABI in a recent PR as it's not a standard ERC-721 function.
Could you please pull master to your branch and update this part?
There is a good example of how we handle `creatorName` in `nifty-gateway.ts` file.